transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDomainSocketChannel.java

```diff
@@ -33,18 +35,18 @@
     private volatile DomainSocketAddress remote;
 
     public EpollDomainSocketChannel() {
-        super(Native.socketDomainFd());
+        super(newSocketDomain());
     }
 
-    public EpollDomainSocketChannel(Channel parent, FileDescriptor fd) {
+    public EpollDomainSocketChannel(Channel parent, Socket fd) {
         super(parent, fd.intValue());
     }
 
     /**
      * Creates a new {@link EpollDomainSocketChannel} from an existing {@link FileDescriptor}
      */
-    public EpollDomainSocketChannel(FileDescriptor fd) {
-        super(fd);
+    public EpollDomainSocketChannel(Socket fd, boolean active) {
```

Comment:
@normanmaurer - I deprecated all the constructors that appear to be misleading and may get users into trouble.